src/Handlers/EditJsonActionHandler.ts

```diff
@@ -45,19 +46,23 @@ export class EditJsonActionHandler implements ActionHandlerContract<'edit-json'>
 
       try {
         Logger.info(`Reading ${file}.`);
-        let content = fs.readJsonSync(targetFile);
+        let content = fs.readFileSync(targetFile).toString();
+        if (content.charCodeAt(0) === 0xFEFF) {
+          content = content.slice(1)
+        }

```

Comment:
Can you explain what is it for?

Reply:
I copied some of this from npm. It also looks like jsonfile (which `fs-extra` uses) does something similar